be/src/vec/exec/volap_scan_node.cpp

```diff
@@ -42,19 +42,16 @@ VOlapScanNode::VOlapScanNode(ObjectPool* pool, const TPlanNode& tnode, const Des
           _tuple_desc(nullptr),
           _tuple_idx(0),
           _eos(false),
-          _max_materialized_row_batches(config::doris_scanner_queue_size),
           _start(false),
           _scanner_done(false),
           _transfer_done(false),
           _status(Status::OK()),
           _resource_info(nullptr),
-          _buffered_bytes(0),
           _eval_conjuncts_fn(nullptr),
           _runtime_filter_descs(tnode.runtime_filters),
           _max_materialized_blocks(config::doris_scanner_queue_size),
           _output_slot_ids(tnode.output_slot_ids) {
     _materialized_blocks.reserve(_max_materialized_blocks);
```

Comment:
_max_materialized_blocks since it is always == config::doris_scanner_queue_size, we could remove this parameter.

Reply:
fixed